samples/Quickstart/README.md

```diff
@@ -92,8 +96,17 @@ This quickstart will create the below resources. These will be used both for loc
 1. Once you are ready to deploy to Azure, we can use azd. Run `azd deploy` from your terminal or command prompt.
 2. The command will output ae endpoint for your function app. Copy this.
 3. Test the endpoint by going to `<Endpoint>/Patient` in your browser or API testing tool.
+4. For APIM endpoint, get APIM Gateway URL from section [Get the deployment details](##get-the-deployment-details) and test endpoint in API testing tool.
+
+## Get the deployment details
+
+ - Run the below command to get the deployed APIM Gateway URL variable named `APIM_GatewayUrl`.
+    ```
+     azd get-values
+    ```
 
 ## Usage details
+#### Quickstart function app
```

Comment:
NIT: This should be three hashes not two. Best not to skip from H2 to H4.

Reply:
Changes to the Quickstart readme document have been made as per review comments.